mypy/main.py

```diff
@@ -700,7 +700,7 @@ def add_invertible_flag(flag: str,
 
     # Set strict flags before parsing (if strict mode enabled), so other command
     # line options can override.
-    if getattr(dummy, 'special-opts:strict'):
+    if getattr(dummy, 'special-opts:strict'):  # noqa
```

Comment:
Curious that this requires a `# noqa` -- apparently flake8 doesn't realize that the literal string is not a valid identifier. :-)